How-To-Release-Riak.md

```diff
@@ -0,0 +1,56 @@
+This doc contains the internal Basho policy for building, testing, and releasing Riak.
+
+# Scheduling
+All public releases must conform to the following schedule:
+* An expected delivery date on a Wednesday. Each delivery date must have at least one fallback date that is no earlier than one week after the expected delivery date.
```

Comment:
@laurenrother yes, I'll fix it in the next iteration.

@javajolt Wednesday would be delivery from Eng, which would give us time to have the go/no-go the next day after we sit on it, and have a day to stage, etc. But yes, standardization, similar to what the TS team is doing